commands/general/macro.js

```diff
@@ -0,0 +1,89 @@
+module.exports.load = (client) => {
+  client.commands['macro'] = {
+    run (message) {
+      let args = message.content.split(' ').splice(1)
+      switch (args[0]) {
+        case 'help':
+            let msg = new client.discord.RichEmbed()
+            msg.setTitle('Macro help')
+            msg.setDescription( 'Heres a list of all available macros' )
+            msg.addField('Commands', `
+                  \`${client.settings.prefix}macro help\` - This message.
+                  \`${client.settings.prefix}macro chat\` - Chat channels information.
```

Comment:
We could have multiple ways of referring to this.

`p!info text`, `p!info text_channels`, `p!info tc` could all be used to trigger the same logic - we'd just need to add it to the switch.

Then the corresponding voice commands would be `p!info voice`, `p!info voice_channels`, and `p!info vc`.